### PR TITLE
Fix inter

### PR DESCRIPTION
**Pull request by garvsl**
fixes the inter font error, this is by having a loading process for when the app is loading and having the font load during that.

### CODE REVIEW

File: app/_layout.tsx [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
+  const [appIsReady, setAppIsReady] = useState(false);
+
+  useEffect(() => {
+    async function prepare() {
+      try {
+        // Pre-load fonts, make any API calls you need to do here
+        await useFonts();
+        // Artificially delay for two seconds to simulate a slow loading
+        // experience.
+        // await new Promise((resolve) => setTimeout(resolve, 2000));
```
Comment (sky159321) [inline app/_layout.tsx:28]: do we want to do this ?

File: hooks/useFonts.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import * as Font from "expo-font";
+
+const useFonts = async () =>
+  await Font.loadAsync({
+    Inter: require("@tamagui/font-inter/otf/Inter-Medium.otf"),
+    InterBold: require("@tamagui/font-inter/otf/Inter-Bold.otf")
```
Comment (sky159321) [inline hooks/useFonts.tsx:6]: Nice

**Review by sky159321 — APPROVED** [review]
NICE APPROVED WITH A QUESTION!!!

File: app/_layout.tsx (hunk 1 above)
Comment (garvsl) [reply app/_layout.tsx:28]: Could be useful for when we're testing. For now I left it as a comment

Comment (sky159321) [reply app/_layout.tsx:28]: :shipit:

**Pull request merged**